man/systemd.resource-control.xml

```diff
@@ -198,6 +198,19 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>CPUIdle=</varname></term>
+
+        <listitem>
+          <para>Takes a boolean argument, defaults to false.
+          When true, mark the cgroup for "idle scheduling", which means that it will get CPU resources only
+          when there are no processes not marked in this way to execute in this cgroup or its siblings.
+          Note that setting CPUWeight is illegal if CPUIdle is true.
```

Comment:
Enclose CPUWeight with `<varname>` and suffix settings with `=`, to clarify, they are seetings, i.e.

`<varname>CPUWeight=</varname>`

Also, the fact that CPUIdle=/CPUWeight= are exclusive to each other and cannot both be used is am artifact of the fact that a cgroup can either be considered of idle or regular scheduling, which is what you should say. 

Moreover afaics it's not actually illegal to set both in the current patch, it's just that the weight has no effect if idle mode is used.